plugins/catalog/package.json

```diff
@@ -33,6 +33,7 @@
     "@backstage/catalog-client": "^0.3.10",
     "@backstage/catalog-model": "^0.7.7",
     "@backstage/core": "^0.7.6",
+    "@backstage/core-api": "^0.2.17",
```

Comment:
`core-api` is internal and should not be used directly, import from `core` instead

Reply:
fixed